src/server/rooms/netsblox-socket.js

```diff
@@ -303,6 +303,7 @@ class NetsBloxSocket {
     }
 
     sendMessageTo (msg, dstId) {
+        if (typeof a !== "string") dstId = dstId.toString();
```

Comment:
what is `a`? It might be better to just put `dstId = dstId + '';  // ensure that it is a string`